gaphor-self-test.txt

```diff
@@ -0,0 +1,7009 @@
+gaphor.ui.selftest INFO System information:
```

Comment:
Hi @pbrown12303, I don't think we want to commit your self-test log to the repository. Maybe we should add it to the gitignore file?

Reply:
I added it to the gitignore file.